_includes/modern-appendix.md

```diff
@@ -585,6 +585,18 @@ Sent as a reply to the {% message MODE %} command, this numeric indicates the en
 
 Sent as a reply to the {% message WHOWAS %} command, this numeric indicates the end of a `WHOWAS` reponse for the nickname `<nick>`. This numeric is sent after all other `WHOWAS` response numerics have been sent to the client.
 
+{% numericheader RPL_INFO %}
+
+      "<client> :<string>"
+
+Sent as a reply to the {% message INFO %} command, this numeric returns information describing the server: e.g. its version, list of authors and contributors, and any other miscellaneous information which may be considered to be relevant.
```

Comment:
Hmm, if this info is only meant to be read by people, could we explicitly specify that here? Just to make 100% sure people don't try to parse what's output by this numeric.

```suggestion
Sent as a reply to the {% message INFO %} command, this numeric returns human-readable information describing the server: e.g. its version, list of authors and contributors, and any other miscellaneous information which may be considered to be relevant.
```